LongWarOfTheChosen/Localization/LW_Overhaul.int

```diff
@@ -372,7 +372,8 @@ m_strInvasionConfirm=VIEW INVASION SITE
 m_strRepeatResearch="REPEAT RESEARCH"
 
 [UIScreenListener_ArmoryWeaponUpgrade_LW]
-strStripWeaponUpgradesButton=Strip Weapon Upgrades
+strStripWeaponUpgradesButton=Strip Upgrades
+StripAllWeaponUpgradesStr =Strip Upgrades From Inactive Soldiers
```

Comment:
Any changes to localisation text should be surrounded by:
```
; LWOTC Needs Translation
...
; End Translation
```
with the '...' representing the changed localisation entries.